### PR TITLE
Implement FromGlib and ToGlib traits on Pid type

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @sdroege @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
Windows is again against us.
Maybe we need use `glib_sys::GPid` instead `i32` ?
```
error[E0308]: mismatched types
  --> src\source.rs:57:9
   |
56 |     fn to_glib(&self) -> i32 {
   |                          --- expected `i32` because of return type
57 |         self.0
   |         ^^^^^^ expected i32, found *-ptr
   |
   = note: expected type `i32`
              found type `*mut libc::c_void`
error[E0308]: mismatched types
  --> src\source.rs:65:13
   |
65 |         Pid(val)
   |             ^^^ expected *-ptr, found i32
   |
   = note: expected type `*mut libc::c_void`
              found type `i32`
```

**Comment by GuillaumeGomez** [comment]
@EPashkin I think too. I'll update the PR.

**Comment by sdroege** [comment]
Wants a `cargo fmt` apparently, otherwise looks good to me.

Maybe not a good idea to run the `cargo fmt` CI test with nightly. That should be done with stable or otherwise we get spurious issues like this one whenever there's a rustfmt regression in nightly.

File: .travis.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ script:
     rustup component add clippy;
     cargo clippy --release;
     fi
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ] && ! [ "$ARM" == "1" ]; then
+  - if ! [ "$TRAVIS_RUST_VERSION" == "nightly" ] && ! [ "$TRAVIS_RUST_VERSION" == "beta" ] && [ "$GTK" == "3.14" ] && ! [ "$ARM" == "1" ]; then
```
Comment (sdroege) [inline .travis.yml]: Hmm why not `== stable`? Hrm, and actually add a stable job to travis in addition to the lower-bound 1.36 job :)

Comment (GuillaumeGomez) [reply .travis.yml]: Sounded simpler that way. I don't want yet another special case.

Comment (sdroege) [reply .travis.yml]: We should have a stable job too though, that is always running with latest stable instead of fixed 1.36

Comment (GuillaumeGomez) [reply .travis.yml]: This is a good argument. Should I run it on both OSX and linux or just linux is fine?

Comment (sdroege) [reply .travis.yml]: all

**Comment by EPashkin** [comment]
👍 @GuillaumeGomez Thanks

**Pull request merged**